charts/trino/values.yaml

```diff
@@ -91,8 +91,9 @@ accessControl: {}
 # accessControl -- [System access
 # control](https://trino.io/docs/current/security/built-in-system-access-control.html)
 # configuration.
+# type can be either 'configmap' when using a local file or 'remote' when using a file from an http/https URL
```

Comment:
All docs should be correct sentences, start with a capital letter.
```suggestion
# Type can be either `configmap` when using a local file, or `remote` when using a file from an URL.
```

Reply:
Also, see the repository README on how to run pre-commit hooks to regenerate the Chart's README.